src/PerfectGym.AutomergeBot/SlackNotifications/SlackMessageProvider.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Text;
+
+namespace PerfectGym.AutomergeBot.SlackNotifications
+{
+    public class SlackMessageProvider : ISlackMessageProvider
+    {
+        private readonly INow _now;
+
+        public SlackMessageProvider(INow now)
+        {
+            _now = now;
+        }
+
+        public string CreateNotifyUserAboutPendingPullRequestMessage(string authorId,
+            IEnumerable<PullRequestDto> pullRequests)
+        {
+            var stringBuilder = new StringBuilder();
+            
+            stringBuilder.AppendLine(
+                $"<@{authorId}> please resolve conflicts manually, approve review and merge pull request");
+            foreach (var pullRequestUrl in pullRequests)
+            {
+                var duration = _now.Now() - pullRequestUrl.CreatedAt;
+                stringBuilder.Append($"PR open for: {duration.Format()}");
+                stringBuilder.AppendLine(pullRequestUrl.Url);
+            }
+
+            return WebUtility.UrlEncode(stringBuilder.ToString());
+        }
+    }
+
+    public static class TimeSpanExt
+    {
+        public static string Format(this TimeSpan duration)
```

Comment:
Moze
```suggestion
        public static string FormatAsDescriptiveForHuman(this TimeSpan duration)
```
?

Samo `Format` nic nie mówi o tym czego możemy się spodziewać po tej metodzie.

Reply:
ok